### PR TITLE
[13.0][MIG] l10n_th_partner

### PR DESCRIPTION
**Pull request by poonlap**
Please revise the migration code for l10_th_partner.

### CODE REVIEW

File: l10n_th_partner/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+* Kitti Upariphutthiphong. <kittiu@gmail.com> (http://ecosoft.co.th)
```
Comment (kittiu) [inline l10n_th_partner/readme/CONTRIBUTORS.rst:1]: You can add yourself as contributor here.
No need to chagnge in README.rst, ocabot will take care.

**Comment by kittiu** [comment]
Please change PR name for easy reading, and to comply with the contribution guideline.
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst
```
[13.0][MIG] l10n_th_partner
```

**Review by kittiu — COMMENTED** [review]
Thank you for your contribution,
hope you enjoy and will do more.
You can also do modules in other OCA repo too. And if you need my involvement, just cc @kittiu in your PR.

**Comment by poonlap** [comment]
> Please change PR name for easy reading, and to comply with the contribution guideline.
> https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst
> 
> ```
> [13.0][MIG] l10n_th_partner
> ```
Thank you for your advice.  PR name was changed.

**Pull request merged**